modules/yieldmoBidAdapter.js

```diff
@@ -176,8 +176,25 @@ export const spec = {
     return bids;
   },
 
-  getUserSyncs: function () {
-    return [];
+  getUserSyncs: function(syncOptions, serverResponses, gdprConsent = {}, uspConsent = '') {
+    const syncs = [];
+    const gdprFlag = `&gdpr=${gdprConsent.gdprApplies ? 1 : 0}`;
+    const gdprString = `&gdpr_consent=${encodeURIComponent((gdprConsent.consentString || ''))}`;
+    const usPrivacy = `us_privacy=${encodeURIComponent(uspConsent)}`;
+    const pbCookieSyncAssistUrl = `https://ads.yieldmo.com/pbcsa?${usPrivacy}${gdprFlag}${gdprString}`;
```

Comment:
nit: Let's make this a const on top like other urls.

Reply:
updated